.gitignore

```diff
@@ -55,7 +55,7 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
+.env.local
```

Comment:
hhm, @lilitkarapetyan  - isn't the `.env.local` the template that we keep in the repo, but `.env` is the one that can get over-written depending upon how we wish to deploy it?